src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -119,8 +121,25 @@ bool mpu6000ReadRegister(uint8_t reg, uint8_t length, uint8_t *data)
     return true;
 }
 
-void mpu6000SpiGyroInit(uint8_t lpf)
+void mpu6000SpiGyroInit(gyro_t *gyro, uint8_t lpf)
 {
+    uint16_t intPeriod;
+    switch(lpf) {
+        case 0:
+            intPeriod = PERIOD_HZ(8000);
```

Comment:
Maybe rename macro to `US_FROM_HZ` ? It will document that `intPeriod` is in uS and that Hz is used here. Using `intPeriod = 125` is bad idea - you need to do inverse mental calculation to realize where the value comes from (gyro update rate is documented in Hz and it's easier to use it in Hz).  `intPeriod = 1000000 / 8000` is possible, but long and does not document units used for `intPeriod`.